src/features/organizations/components/MemberManageModal.tsx

```
@@ -0,0 +1,281 @@
+      setMembers(list);
+    } catch (err) {
+      setError(
+        err instanceof Error ? err.message : "멤버 목록을 불러오지 못했습니다.",
+      );
+    } finally {
+      setLoading(false);
+    }
+  }, [organization.id]);
+
+  useEffect(() => {
+    void fetchMembers();
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/features/organizations/components/MemberManageModal.tsx:77:10
  75 |
  76 |   useEffect(() => {
> 77 |     void fetchMembers();
     |          ^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  78 |   }, [fetchMembers]);
  79 |
  80 |   const handleInvite = async () => {

(react-hooks/set-state-in-effect)